crates/autopilot/src/run_loop.rs

```diff
@@ -289,19 +274,27 @@ impl RunLoop {
             return;
         }
 
-        observe::unsettled(&solutions, &winners, &auction);
-        for Participant { driver, solution } in winners {
-            tracing::info!(driver = %driver.name, solution = %solution.id(), "winner");
+        // Mark all non-winning orders as `Considered` for execution
+        self.persistence.store_order_events(
```

Comment:
This change is not equivalent since `non_winning_orders()` can actually return order ids that are also contained in winning orders.

Maybe it's easiest to have one extra function that handles `Considered` and `Executed` together. It could first loop over the winners  and store the order ids in the winning bucket and then iterate over the losers and put the order ids into the loser bucket (iff the same id was not already part of a winner bucket).

Seems reasonable to have this as part of the post processing step. 🤔

Reply:
> This change is not equivalent 

Ah true, was moving fast and didn't think it through well enough. I fixed it and kept it in the main loop just because there is also another READY state at the beginning, so I wanted to keep them all at the same level (although I am also ok with moving each of them into separate function for readability).

> Seems reasonable to have this as part of the post processing step

Maybe, but I want postprocessing function removed completely once we move to new db tables, so we will just have a few `self.persistence` function calls instead (save auction, save competition and save fee policies).